features/SPS002_Correct_spatial_breakdown.feature

```diff
@@ -4,8 +4,8 @@ Feature: SPS002 - Correct spatial breakdown
 The rule verifies that spatial elements are aggregated as per the Spatial Composition Table.
 The possible allowed breakdown can be found in the csv file in the folder 'features/resources/spatial_CompositionTable.csv'
 
-  Scenario: Agreement on each IfcSpatialElement being aggregated as per spatial composition table.
+  Scenario: Agreement on each IfcSpatialElement and IfcProject being aggregated as per spatial composition table.
 
       Given A file with Schema Identifier "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3" or "IFC4"
-      And An IfcSpatialElement
-      Then It must be aggregated as per spatial_CompositionTable.csv
\ No newline at end of file
+      And An IfcObjectDefinition
+      Then It must be aggregated as per spatial_CompositionTable.csv
```

Comment:
We can't do that anymore, because `be aggregated as per {X}` now looks up both directions, i.e both IsDecomposedBy and Decomposes. Do you think this is too unclear in the current wording?

Reply:
> looks up both directions

Oh, that wasn't clear to me. It can remain that way.

> IsDecomposedBy is not empty

I'm just wondering if the rule is not requiring *all* `IfcObjectDefinition` to be aggregated?

Reply:
> I'm just wondering if the rule is not requiring all IfcObjectDefinition to be aggregated?

See note above:

> - When you have a child it **must** have the correct parent. Assigning an IfcBuildingStorey to a IfcRailway is forbidden. An IfcBuildingStorey not assigned to anything is also forbidden.
> - When you have a parent any of its children **must** be according to the rules, but you can have 0 children. You can have an IfcBuilding without IfcBuildingStoreys.